### DISCUSSION TITLE
Github notifications

### DISCUSSION THREAD

**Discussion by allenjd3** [community/community#17379] [Other Feature Feedback, Questions, & Ideas]
I’ve been having a consistent 500 error on the GitHub notifications route since yesterday. Is anyone else having this issue?

**Comment by queenofcorgis** [comment]
👋🏼 

Checking in here - are you still experiencing this?

**Reply by allenjd3** [reply]
It has been resolved thanks (sorry for late response, I've been off for a few weeks)

**Reply by queenofcorgis** [reply]
No worries - thanks for updating us and glad this is resolved 🎉

**Reply by 007lva** [reply]
I'm also having 500 errors with my current job's organization notifications, sometimes also with some open source organizations, like [dry-rb](https://github.com/notifications?query=org%3Adry-rb).
